modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -549,6 +736,43 @@ AUI.add(
 						'#' + inputPlaceholder.attr('id') + '_desc'
 					);
 					instance._flags = boundingBox.one('.palette-container');
+
+					var activeLocales = instance.get('activeLocales');
+
+					if (activeLocales) {
+						instance._flagsInitialContent = instance._flags.cloneNode(
+							true
+						);
+
+						instance._renderActiveLocales();
+
+						Liferay.Loader.require(
+							[
+								'@liferay/frontend-js-state-web@1.0.6/index',
```

Comment:
@izaera is there any way we can get rid of the versions in these modules loading?

Reply:
Do I have a deja vu? (https://liferay.slack.com/archives/C3JBR21HA/p1624274153186200)

Reply:
Is there any way I don't need to read a thread of 140 messages? xDDD

Reply:
I don't think it even has your answer 😂 ... I can investigate this case but pretty sure @izaera knows it directly.

Reply:
One thing to keep in mind (other than the version numbers) is that this approach creates an important race condition. It turns an otherwise sync method into an async one. This can cause some hard-to-debug issues in the future depending on what and when we expect code to run.

Reply:
We've moved the loading of the modules out to the jsp so the AUI component is not loaded until the AMD module are loaded too, so when we instantiate the component everything is loaded and shouldn't be any race condition. Sounds good?

Reply:
> Sounds good?

![nod](https://user-images.githubusercontent.com/905006/130061318-7ce2222f-c608-4840-a290-340d98b0e00b.gif)